mantidproject/mantid
# Plot parameter against axis1.

Fixes #14336

**To test**
1. Load a workspace (eg MUSR00015189)
2. Add all spectra from the workspace to the multi-dataset fitting interface.
3. Add a function (eg ExpDecay)
4. Select Sequential fitting.
5. Select a Plot parameter
6. Leave Log Value empty
7. Fit
8. Check that a plot is shown in a new window. The plot should be of the Plot Parameter against the spectrum number (or whatever value is on the axis 1).
